SEED-platform/seed
# Update the update datetime field when changing tax lot or property

#### Any background context you want to provide?
When manually updating a property or taxlot would not update the updated date time.

#### What's this PR do?
* When manually changing a field via the UI, then make sure the update datetime changes.

#### How should this be manually tested?
* Upload data
* Select a random property and view the detailed page
* Edit any of the values in the detailed page
* Verify that the date time changed to most recent timestep
* Do it again on the same data and make sure it still works (the second time is actually a slightly different path through the code)
* Repeat the whole procedure for a tax lot

#### What are the relevant tickets?
N/A

#### Screenshots (if appropriate)
N/A

#### Definition of Done:
